### PR TITLE
Feat/add creating steps and other modifications

### PR DESCRIPTION
**Pull request by NatLee**
- Add step-by-step guidance for creating new tunnels.
- Modify the style of `Telepy`.
- Update the log page.

### CODE REVIEW

**Comment by NatLee** [comment]
Log page
image

File: src/reverse_keys/views.py [hunk 1]
```diff
@@ -94,7 +95,7 @@ def post(self, request, token, *args, **kwargs):
         if not host_friendly_name[0].isalpha():
             return Response({"error": "Host friendly name cannot start with a number."}, status=status.HTTP_400_BAD_REQUEST)
 
-        key:str = request.data.get('key', None)
+        key:Optional[str] = request.data.get('key', None)
```
Comment (h-alice) [inline src/reverse_keys/views.py:98]: I used to abuse `Union[str, None]` and here's a better solution :D.

File: src/static/js/logs.js
Comment (h-alice) [inline src/static/js/logs.js:1]: JavaScript :(

File: src/tunnels/views.py [hunk 2]
```diff
@@ -22,7 +22,8 @@
 # ========================================
 """
 
-CLIENT_NO_USER_STEM = """# ========================================
```
Comment (h-alice) [inline src/tunnels/views.py:25]: \n :D

**Review by h-alice — APPROVED** [review]
Currently no discovered issues, let's go 🚀

File: src/static/js/logs.js
Comment (NatLee) [reply src/static/js/logs.js:1]: JS sucks :'(

File: src/tunnels/views.py (hunk 2 above)
Comment (NatLee) [reply src/tunnels/views.py:25]: Make here more pretty with `\n`. :D

image

Comment (NatLee) [reply src/tunnels/views.py:25]: > It's in the `config` part with every tunnel row.

**Comment by NatLee** [comment]
Let'ssssssssss GOOOOOOOOO!

**Pull request merged**